src/server/debug/server/server.go

```diff
@@ -39,22 +43,25 @@ const (
 	pachdPrefix     = "pachd"
 	pipelinePrefix  = "pipelines"
 	podPrefix       = "pods"
+	databasePrefix  = "database"
 )
 
 type debugServer struct {
 	env           serviceenv.ServiceEnv
 	name          string
 	sidecarClient *client.APIClient
 	marshaller    *jsonpb.Marshaler
+	database      *pachsql.DB
 }
 
 // NewDebugServer creates a new server that serves the debug api over GRPC
-func NewDebugServer(env serviceenv.ServiceEnv, name string, sidecarClient *client.APIClient) debug.DebugServer {
+func NewDebugServer(env serviceenv.ServiceEnv, name string, sidecarClient *client.APIClient, db *pachsql.DB) debug.DebugServer {
 	return &debugServer{
 		env:           env,
 		name:          name,
 		sidecarClient: sidecarClient,
 		marshaller:    &jsonpb.Marshaler{Indent: "  "},
+		database:      db,
```

Comment:
maybe I missed something but why can't we extract db from env.GetDBClient() here?

Reply:
I think we would mock env's GetDBClient in that case. I don't think we should expand the signature here

Reply:
Yeah we have a serviceenv that works for testing, so in theory not necessary, but I think less coupling is better in general.  We already have the problem where the serviceenv can't actual return built servers (because of a dependency cycle).  If the servers were built by the serviceenv instead of building themselves from a serviceenv, we wouldn't have that problem.